#!/usr/bin/python3
from shutil import ExecError


def raise_exception_msg(message=""):
    raise NameError(message)
